src/app/pages/common/entity/entity-table/entity-table.component.ts

```diff
@@ -241,7 +241,13 @@ export class EntityTableComponent implements OnInit, AfterViewInit {
     for (let i = 0; i < rows.length; i++) {
       for (const attr in rows[i]) {
         if (rows[i].hasOwnProperty(attr)) {
-          rows[i][attr] = this.rowValue(rows[i], attr);
+          if (rows[i][attr] === true) {
+            rows[i][attr] = 'yes';
```

Comment:
If we use "yes" and "no", I think a T() call is needed.  But checkboxes would be (or should be) consistent with the edit forms and don't need translation at all.

Reply:
Use translate.get('yes') to accomplish the translated effect, if you need help finding that let me know.